File: Server/Spell.js
module.exports = class Spell {
    constructor() {
        this._id = '';
        this.name = '';
        this.actionPointsConsuption = 0;
        this.damage = 0;
        this.range = 0;
        this.explosiveRange = 0;
        this.cooldown = 0;
    }
}

File: Assets/Scripts/Utils/Spell.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

[System.Serializable]
public class Spell
{
    public string _id;
    public string name;
    public float actionPointsConsuption;
    public float damage;
    public float range;
    public float explosiveRange;
    public float shield;
    public float shieldDuration;
    public float cooldown;
    public string animationKind;

    public Sprite icon;
    public int m_turnCooling;
}
